frontend/src/metabase/public/lib/code-templates.ts

```diff
@@ -1,16 +1,38 @@
 import type {
   CodeSampleParameters,
   EmbeddingDisplayOptions,
-  EmbeddingParameters,
+  EmbeddingParametersValues,
 } from "./types";
 import { optionsToHashParams } from "./embed";
 
+function getIframeQuerySource(displayOptions: EmbeddingDisplayOptions) {
```

Comment:
Extract this into a function because every language basically use the same function. So I could just add some custom logic here.